### ISSUE TITLE
Are LEGOs getting more expensive? A brick by brick analysis!

### ISSUE THREAD

**Issue by nickospi** [jsoma/data-studio-projects#166]
# Pitch
Newsweek celebrates 60 years of LEGO with a special issue. As there is a constant discussion that LEGOs are getting more and more expensive, I set off to find out the price of the LEGO brick through the years.
![nw_lego_cover_noupcspine_1024x1024](https://user-images.githubusercontent.com/39416389/42823388-6321f0e8-89ab-11e8-8db2-111d02777560.jpg)

[Brickset.com](http://www.brickset.com) is the most established, accurate and up-to-date online LEGO database today: it contains thorough information on 15089 LEGO sets and other items released over the last 69 years.
I contacted them and they generously offered their complete dataset for this project.
Apart from this dataset, I also collected [data on inflation](https://www.federalreserve.gov) by the Federal Reserve Bank  as well as [other LEGO datasets](https://www.kaggle.com/rtatman/lego-database) on colors, spare parts and themes.
## Details
Possible headline(s): How much does a LEGO brick cost? A brick by brick analysis!

Data set(s): Every LEGO set out there with pricing, color, parts, year and more and more data, USD inflation rates data.

Code repository: https://github.com/nickospi/data_studio/tree/master/code/Lego

Possible problems/fears/questions: NaN

## Work so far

I've cleaned up the data sets, did a first analysis on mean prices per year and added an inflation adjustment column. 
### Graph 1: Average price of LEGO sets per year
![lego_prc](https://user-images.githubusercontent.com/39416389/42796439-1ec622a0-8958-11e8-916c-edda807f55b6.png)

Nice to have :
- A graph showing price per brick per year
- A graph showing number of sets per year
- A graph showing number of bricks per set Theme/Year
- Something with LEGO colors per set Theme/Year

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue issue has `[Project]` in the title
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by nickospi** [comment]
# Update
    	
    ## Project content: images/words/etc
Worked with annotations on Adobe Illustrator. 
    	   	
    
![lego_prc1](https://user-images.githubusercontent.com/39416389/42978365-7d84b04c-8b9a-11e8-8855-f5951845f436.png)

________________________________________________________________________________________________

![lego_prc2lines](https://user-images.githubusercontent.com/39416389/42978371-83654f94-8b9a-11e8-9c8b-24ca4b71f582.png)

      
    ## Any changes in direction or topic?
      
Nan
    
 ## Problems/Questions
- The main issue up until now has been to adjust prices for inflation. 
- I have to work more on the second graph, comparing median and mean prices and also move on to the next steps by using more data.       
- Next steps:  a graph showing price per brick per year , a graph showing number of sets per year, a graph showing number of bricks per set Theme/Year, something with LEGO colors per set Theme/Year -I'd better get back to work!
   	
    ## Checklist
    	
    - [X] I have included my visuals
    - [X] I have filled out the sections above
    - [X] I have been updating my DIARY.md with details about my process
    - [X] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Howdy! I'm a little robot, let's see what's been going on here.

You need some feedback, let me summon @linleysanders, @SiruiZhu, @dz2383 for you

It looks like we need to **fix up your [your update](https://github.com/jsoma/data-studio-projects/issues/166#issuecomment-406462366)** a little bit! Edit it by clicking the pencil in the top right-hand corner. It requires:

* New, updated content for your project
* The checklist (be sure you're using the template!)

Maybe you just didn't use [the template](https://github.com/jsoma/data-studio-projects/blob/master/UPDATE_TEMPLATE.md)? If not, edit your comment, cut and paste the template in, and then fill it out.

**Comment by linleysanders** [comment]
I really like your Lego chart overall—I think the headline should reflect your conclusion, like "LEGO is getting more expensive as xxx," with the subhead noting its peak price took place in 1968.

Are you still planning on having the other charts? I could see this one standing alone well!

**Comment by SiruiZhu** [comment]
I like your "Lego"-color scheme!! and I like the way you highlighted the numbers. 

I don't know if that's the case, for lego set there are some really expensive sets and some lego sets are small and not so expensive(relatively), it might be better to just compare the median?

**Comment by dz2383** [comment]
Love your topic and the color scheme! Maybe the headline could be bigger and be a statement?

**Comment by nickospi** [comment]
# Final

# Project visuals/text
All these make much more sense [here](https://nickospi.github.io/) 

If you prefer the standalone version, it's this one:

### Average price of a kit/year, adjusted for inflation:

![lego1](https://user-images.githubusercontent.com/39416389/43368621-2bbbf856-932e-11e8-893f-e6be37b8143f.png)

### Average and median price of a kit/year, adjusted for inflation:
![lego2](https://user-images.githubusercontent.com/39416389/43368622-2e198a1e-932e-11e8-9fa9-8ed98f6af2b2.png)

### Most expensive sets ever

![lego3](https://user-images.githubusercontent.com/39416389/43373302-c788657e-9376-11e8-8264-4c52c2dc8907.png)




### Most bricks per set per decade (these are normalised)
![lego4_2](https://user-images.githubusercontent.com/39416389/43368624-3331be40-932e-11e8-98b5-94341215e8ab.png)

### Top 30 Themes with the most bricks ever in two versions (one less kitsch than the other)

![legowthlogos](https://user-images.githubusercontent.com/39416389/43371421-a81147f4-935f-11e8-8446-732af4e4c9e9.png)

![legowthnames](https://user-images.githubusercontent.com/39416389/43368627-38c447ec-932e-11e8-84f2-c0011fd56917.png)

## Details

**Headline:** 
Are LEGOs getting more and more expensive? A brick by brick analysis!
**Published website version:**
https://nickospi.github.io/Lego

**Code repository:**
https://github.com/nickospi/data_studio/tree/master/code/Lego

**Final data set(s):** 
https://github.com/nickospi/data_studio/blob/master/code/Lego/all_brickset.csv
https://github.com/nickospi/data_studio/blob/master/code/Lego/infl.csv

## What did you find to be the most difficult part of this project?
Adjusting prices for inflation. Manipulating a large database. Making graphs about colorful toys and logos without looking too kitsch. Officially stopping for now this project as I like LEGO a lot.  
## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I would do more analysis with that stuff if I had the time. 

## Checklist

- [X] I have included my visuals
- [X] I have posted my project to my project website
- [X] I have been updating my DIARY.md with details about my process
- [X] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
